design-system/materials/internals/definition.yaml

```diff
@@ -323,6 +445,87 @@ decisionGroupsByTheme:
           description: ''
           choice: color-neutral-95
 
+    fontSizes:
+      label: Font Sizes
+      prefix: font-size
+      decisions:
+        font-size-for-input:
+          description: ''
+          choice: font-size-30
+        font-size-for-text-as-h1:
+          description: ''
+          choice: font-size-50
+        font-size-for-text-as-h2:
+          description: ''
+          choice: font-size-50
+        font-size-for-text-as-h3:
+          description: ''
+          choice: font-size-40
+        font-size-for-text-as-h4:
+          description: ''
+          choice: font-size-30
+        font-size-for-text-as-h5:
+          description: ''
+          choice: font-size-30
+        font-size-for-text-as-body:
+          description: ''
+          choice: font-size-30
+        font-size-for-text-as-detail:
+          description: ''
+          choice: font-size-10
+
+    lineHeights:
+      label: Line Heights
+      prefix: line-height
+      decisions:
+        line-height-for-text-as-h1:
+          description: ''
+          choice: line-height-50
+        line-height-for-text-as-h2:
+          description: ''
+          choice: line-height-50
+        line-height-for-text-as-h3:
+          description: ''
+          choice: line-height-30
+        line-height-for-text-as-h4:
+          description: ''
+          choice: line-height-20
+        line-height-for-text-as-h5:
+          description: ''
+          choice: line-height-20
+        line-height-for-text-as-body:
+          description: ''
+          choice: line-height-40
+        line-height-for-text-as-detail:
+          description: ''
+          choice: line-height-10
+
+    fontWeights:
+      label: Font Weights
+      prefix: font-weight
+      decisions:
+        font-weight-for-text-as-h1:
+          description: ''
+          choice: font-weight-500
+        font-weight-for-text-as-h2:
+          description: ''
+          choice: font-weight-500
+        font-weight-for-text-as-h3:
+          description: ''
+          choice: font-weight-500
+        font-weight-for-text-as-h4:
+          description: ''
+          choice: font-weight-500
+        font-weight-for-text-as-h5:
+          description: ''
+          choice: font-weight-500
+        font-weight-for-text-as-body:
+          description: ''
+          choice: font-weight-400
+        font-weight-for-text-as-detail:
+          description: ''
+          choice: font-weight-400
+
```

Comment:
I think the case for "bold" text is missing, isn't it?

Reply:
In the board we're using as reference we didn't see any reference to `bold` weight so we assumed we will still using `700` in the new theme so no token would be needed for that.

Perhaps @FilPob might confirm this point.

Reply:
I think the board is missing the bold styles.